### PR TITLE
Teaching together

### LINKED ISSUE #593: providing guidance for helpers
Several conversations have been raised about how we bring helpers into workshops, how we prepare them to follow our pedagogical practices and make them aware of the CoC. We should add something explicit about this to instructor training and update our template emails for helper recruitment to include this info.

### LINKED ISSUE #664: Recommendations for co-instructors
How to be present/help your fellow instructors!  Maybe add to instructor notes?

### PR DESCRIPTION
**Pull request by gvwilson**
Explain how to teach with a partner.
- Closes #664.
- Closes #593 (maybe).

### CODE REVIEW

File: _episodes/11-practice-teaching.md [hunk 1]
```diff
@@ -228,13 +229,107 @@ may have a fixed mindset about teaching. By strengthening your growth mindset wi
...
+  time for a bathroom break" are all useful.
+
+- Each person should teach for at least 10-15 minutes at a stretch,
+  since students may be distracted by more frequent interleaving.
+
+- The person who isn't teaching shouldn't interrupt, offer
+  corrections, elaborations, or amusing personal anecdotes, or do
+  anything else to distract from what the person teaching at the time
+  is doing or saying.  The one exception is that it's sometimes
+  helpful to ask leading questions, particularly if the learners seem
```
Comment (lexnederbragt) [inline _episodes/11-practice-teaching.md]: Non-native English speakers may not know what a leading question is. Maybe use "helpful to ask questions that may be in the minds of several learners" (not sure if that is a good way to phrase it)

**Comment by gvwilson** [comment]
Could someone please merge or close.

**Comment by arokem** [comment]
Looks like this hasn't been addressed: https://github.com/carpentries/instructor-training/pull/671/files#r182372081

Also, what's the status of #718?

File: _episodes/11-practice-teaching.md (hunk 1 above)
Comment (gvwilson) [reply _episodes/11-practice-teaching.md]: Wording changed.

**Comment by ChristinaLK** [comment]
@gvwilson, I did a pretty heavy edit of this and re-submitted. 
#718
Can you review?

**Pull request closed without merging**